addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -183,6 +184,7 @@ export class Chatbot extends Record {
     restart() {
         if (this.currentStep) {
             this.currentStep.isLast = false;
+            this.thread.livechat_active = true;
```

Comment:
Can't we just rely on what's done by the server? Local state should rely on record insert stuff as much as possible, modifying it in place is error prone.